client/signup/config/steps.js

```diff
@@ -154,4 +154,14 @@ export default {
 		providesDependencies: [ 'siteId', 'siteSlug', 'domainItem', 'themeSlugWithRepo' ],
 		delayApiRequestUntilComplete: true
 	},
+	'site-picker': {
+		stepName: 'site-picker',
+		apiRequestFunction: stepActions.linkExistingSite,
+		props: {
+			headerText: i18n.translate( 'Choose your site?' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Choose a site' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).